rebuild-wekan.bat

```diff
@@ -1,10 +1,5 @@
 @ECHO OFF
 
-REM NOTE: You can try this to install Meteor on Windows, it works:

```

Comment:
These changes are in latest Wekan git repo, added by me, you should not remove them. You should check that you are up-to-date with latest changes at git repo.